### DISCUSSION TITLE
Explanation of carbon intensity estimates

### DISCUSSION THREAD

**Discussion by potash** [electricitymaps/electricitymaps-contrib#4963] [Q&A] [answered]
I am looking at two grids in Texas right now ("2023-01-21 11:03:03 UTC") and confused by the estimated carbon intensities:

- The "Electric Reliability Council Of Texas, Inc." is reported to be about 66% gas and 33% coal at the moment, with estimated carbon intensity of 379 g/kWh.
- The "El Paso Electric Company" is reported to be almost 100% gas at the moment, but the carbon intensity is estimated to be 656 g/kWh.

I would assume that gas is less intensive than coal so the intensities would be switched. Can someone explain these estimates to me?

PS This is a fantastic resource. Thanks to everyone who contributed.

**Comment by VIKTORVAV99** [comment, marked as answer]
Not sure where you got the 66% gas 33% coal figures from, as far as I can tell ERCOT has a 13% nuclear, 10.57% coal, 29.28% wind and 46.97% gas share at 2023-01-21T11:00Z.

Nuclear and wind have a significantly lower carbon intensity than both coal and gas so that is what gives it the lower carbon intensity.

**Reply by potash** [reply]
Ah, I was accidentally looking at the "carbon emissions" tab instead of "electricity consumption." Thanks for the quick reply!

Follow up question: what is the source for the pumped hydro storage carbon intensity and how localized is it? I am seeing 15.6 t/min in the New York ISO.

**Reply by VIKTORVAV99** [reply]
No problem!

As for the hydro storage it should be the average carbon intensity per zone per year. So should be different for each region. Ff there is no historical data for a region I think it uses the world average until we have a years worth of historical data.
